github/src/main/kotlin/gropius/Application.kt

```diff
@@ -1,41 +1,61 @@
 package gropius
 
-import gropius.sync.github.Incoming
+import gropius.sync.github.SyncSelector
 import io.github.graphglue.data.repositories.EnableGraphglueRepositories
 import kotlinx.coroutines.runBlocking
+import org.neo4j.driver.Driver
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.CommandLineRunner
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.context.event.ApplicationReadyEvent
 import org.springframework.boot.runApplication
-import org.springframework.context.ApplicationContext
 import org.springframework.context.ConfigurableApplicationContext
+import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.event.EventListener
 import org.springframework.data.mongodb.repository.config.EnableReactiveMongoRepositories
+import org.springframework.data.neo4j.core.ReactiveDatabaseSelectionProvider
+import org.springframework.data.neo4j.core.transaction.ReactiveNeo4jTransactionManager
 import kotlin.system.exitProcess
 
 /**
- * @param incoming Reference for the spring instance of Incoming
  * @param configurableApplicationContext Reference for the spring instance of ConfigurableApplicationContext
  */
 @Configuration
 class SyncApplication(
```

Comment:
class name is misleading, maybe rename to `SyncConfiguration`?

Reply:
0e80459f20b998c4ffb0c7bb90800625b38d3af6